.bazelrc

```diff
@@ -47,3 +47,7 @@ build --host_force_python=PY2
 run --host_force_python=PY2
 
 common --experimental_allow_incremental_repository_updates
+
+
+# Disable Test Caching
+test --nocache_test_results
```

Comment:
Why do we need to disable test caching?

Reply:
Some of the test suits are affecting each other due to resource usages or something else. When a test doesn't run along with the affected test, it passes. That's why we want to run tests every time to ensure tests are passing together.